global.R

```diff
@@ -66,9 +66,18 @@ ordered_tabs <- c(
   "report"
 )
 
+userData <- list()
+if (file.exists("secret.yml")) {
+  secret_pars <- read_yaml("secret.yml")
```

Comment:
again can we replace pars with something else.

Reply:
renamed to "settings"